src/components/Todos/TodosFooter.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, memo, useContext } from 'react';
+import { TodosContext } from './TodosContext';
+
+export const TodosFooter: FC<{ todosCount: number }> = memo(({
+  todosCount,
+}) => {
+  const {
+    setTodos,
+    filterType,
+    setFilterType,
+  } = useContext(TodosContext);
+
+  return (
+    <div className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${todosCount} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={`filter__link ${filterType === 'all' ? 'selected' : ''}`}
+          onClick={() => setFilterType('all')}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={`filter__link ${filterType === 'active' ? 'selected' : ''}`}
+          onClick={() => setFilterType('active')}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={`filter__link ${filterType === 'completed' ? 'selected' : ''}`}
```

Comment:
classnames

Reply:
I have the best implementation in the second task. I went 1st, 3rd, 2nd

So please understand after switching from 2 to 3. 

1. react_todo-app-loading-todos (this one)
2. React Todo App Add and Delete (lastest version)
3. React Todo App with API

Reply:
@UmizDemud i'd appreciate you to walk me through it